cdap-ui/app/features/workers/templates/tabs/runs.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <div class="col-xs-12">
+    <strong> View Run: </strong>
+    <div class="btn-group dropdown-left"
+          dropdown is-open="dropdown.isopen">
+      <button type="button"
+              class="btn btn-default dropdown-toggle"
+              dropdown-toggle ng-disabled="current === 'No Run'">
+        {{ runs.selected.runid }}
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu" role="menu">
+        <li ng-repeat="run in runs">
+          <a ui-sref="worker.detail.runs.run({runid: run.runid})">{{ run.runid }}</a>
+        </li>
+      </ul>
+    </div>
+
+  </div>
+</div>
+<br/>
+<div ng-if="runs.length">
+  <div ng-include="'/assets/features/workers/templates/tabs/runs/run-detail.html'"
+        ng-if="!$stateParams.runid">
+  </div>
+  <div ui-view  ng-if="$stateParams.runid"> </div>
+</div>
+<div ng-if="!runs.length">
+  <div class="well well-lg text-center h3">
```

Comment:
Might not be in this PR but we should probably not use h3 class here. Seems to be a hacky way of adding some margin on the top of a well.. we should probably have the tabs-content to have a padding top and have all children to use that instead of each children having a margin top.